domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/MediationRow.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import lombok.Builder;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Builder
+public class MediationRow {
+    @Builder.Default
+    private String siteId = "4";
+    @Builder.Default
+    private String caseType = "1";
+    @Builder.Default
+    private String checkList = "4";
+    @Builder.Default
+    private String partyStatus = "5";
```

Comment:
If these won't be changed, make them `final`?

Reply:
done

Reply:
changed logic slightly but using final now